README.md

```diff
@@ -102,6 +102,7 @@ srcd init
 # You can also provide a path
 srcd init /home/user/replace/path/
 ```
+**Note:** Ensure that you initialize the source{d} engine everytime you want to process a new repository. Because source{d} engine does not automatically detect the changes in the `init` folder.
```

Comment:
It feels a bit awkward to have too many "source{d} Engine" close together. I propose:
```suggestion
**Note:** Ensure that you initialize source{d} Engine every time you want to process a new repository. Changes in the `init` working directory are not detected automatically.
```